app/src/context/theme.ts

```diff
@@ -0,0 +1,115 @@
+import { useCallback } from "react";
+import { darkColors, lightColors } from "@fuel-ui/css";
+import { useFuelTheme } from "@fuel-ui/react";
+
+interface ColorMapping {
+  light: string;
+  dark: string;
+}
+export type ColorName =
+  | "chrome"
+  | "black1"
+  | "white1"
+  | "white2"
+  | "white3"
+  | "white4"
+  | "gray1"
+  | "gray2"
+  | "gray3"
+  | "gray4"
+  | "gray5"
+  | "gray6"
+  | "gray7"
+  | "gray8"
+  | "sgreen1"
+  | "disabled1"
+  | "disabled2"
+  | "disabled3";
+
+const COLORS: Record<ColorName, ColorMapping> = {
+  chrome: {
```

Comment:
nit: this isn't actually a color, it's the name of the editor theme. It might be better to return `editorTheme` from `useTheme` so it's clear that it's distinct / that it shouldn't be used as a CSS color.

```
export default function useTheme() {
// ...
  const editorTheme = useMemo(
    () => currentTheme === "light" ? "chrome" : "tomorrow_night_bright",
    [currentTheme],
  );
  return {
    theme: currentTheme,
    editorTheme, ...
// ...
```